### DISCUSSION TITLE
How to accumulate result of matrix multiplication to two destination matrices rather than one?

### DISCUSSION THREAD

**Discussion by dyhes** [NVIDIA/cutlass#1361] [Q&A]
I am trying to implement strassen's fast matrix multiplication algorithm based on cutlass, and could not figure out how to accumulate result of matrix multiplication to two destination matrices.
Could someone help me? Thank you in advance!
Two related pictures：
<img width="927" alt="image" src="https://github.com/NVIDIA/cutlass/assets/73648326/ebeb71dc-5007-4130-bdf9-ace1007a3b85">
![WeChat3faacc4ad9505a0b2edd443358dd6eb7](https://github.com/NVIDIA/cutlass/assets/73648326/82220ece-e6be-4b09-ada1-deacd0a314aa)

**Comment by thakkarV** [comment]
at what granularity do you want to do this? A single instruction? After the whole tile wide GEMM during the epilogue?

**Reply by dyhes** [reply]
sorry for late reply
I am wondering if I can do this in the epilougue

**Comment by hwu36** [comment]
related.  https://dl.acm.org/doi/10.1145/3372419  @jianyuh

**Reply by dyhes** [reply]
sorry for late reply
Yes, it is one of the papers I based on. But none of the papers provide any code.
